src/Query.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL;
+
+use Doctrine\DBAL\Types\Type;
+
+/**
+ * A SQL query together with its bound parameters.
```

Comment:
It should be an "An SQL" (because /ˌɛsˌkjuːˈɛl/) for consistency with the rest of the documentation.

Reply:
Right, done in https://github.com/doctrine/dbal/pull/4387/commits/0c840ce893aced99b093d2bb0be55d7e124cdad8.